quizApp.js

```diff
@@ -222,18 +221,50 @@ function doSorting(type) {
 const showbtn = document.querySelector("#show");
 showbtn.addEventListener("click", showAllQuestions);
 
-function showAllQuestions() {
-  const section = document.getElementById("quiz-questions");
-  section.innerHTML = "";
-  fetch(
-    "https://raw.githubusercontent.com/Saikiruthiga/Saikiruthiga.github.io/main/sample/quiz.json"
-  )
-    .then((res) => res.json())
-    .then((question) => displayQuestion(question, true))
-    .catch((error) => console.log(error))
-    .finally(() => console.log("done"));
+async function showAllQuestions() {
+  try {
+    const section = document.getElementById("quiz-questions");
+    section.innerHTML = "";
+    const res = await fetch(
+      "https://raw.githubusercontent.com/Saikiruthiga/Saikiruthiga.github.io/main/sample/quiz.json"
+    );
+    const data = await res.json();
+    displayQuestion(data, true);
+
+    const submitQuizBtn = document.getElementById("submit-quiz");
+    submitQuizBtn.removeAttribute("hidden");
+    submitQuizBtn.addEventListener("click", calcScore);
+  } catch (error) {
+    console.log(error);
+  }
+
   displayQuestion(questionArray, true);
 }
+function calcScore() {
+  let score = 0;
+  const questions = document.querySelectorAll("#quiz-questions > div");
+  questions.forEach((question) => {
```

Comment:
Try adding empty lines to make the code easier to read :)

Reply:
Sure, I keep this in mind.